app/controllers/pages_controller.rb

```diff
@@ -22,13 +22,17 @@ def create
     @page = PageBuilder.create( page_params )
 
     if @page.valid?
-      redirect_to edit_page_path(@page)
+      redirect_to edit_page_path(@page.id)
```

Comment:
I've explicitly passed the `id` so campaigners edit pages at `/pages/1/edit` and not `/pages/foo-bar/edit`